inc/server/class-form-server.php

```diff
@@ -7,6 +7,8 @@
 
 namespace ThemeIsle\GutenbergBlocks\Server;
 
+use PHP_CodeSniffer\Reports\Json;
```

Comment:
Why exactly? PHP code sniffer is a dev-dependency so it won't be present in production. Have you tested to make sure no error is thrown there?

Reply:
I think this was added automatically by VsCode, no idea why is there.

Reply:
You somewhere tried to use JSON as a class and VS code automatically added it, I suppose. :D